pages/common/doctl-apps.md

```diff
@@ -0,0 +1,36 @@
+# doctl apps
+
+> Used to manage digitalocean apps.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/apps>.
+
+- Create an app:
+
+`doctl apps create`
+
+- Create a deployment:
+
+`doctl apps create-deployment`
+
+- Delete an app interactively:
+
+`doctl apps delete {{app_id}}`
+
+- Get an app:
+
+`doctl apps get`
+
+- List all apps:
+
+`doctl apps list`
+
+- List all deployments:
+
+`doctl apps list-deployments`
```

Comment:
Based on docs the app id is required. https://docs.digitalocean.com/reference/doctl/reference/apps/list-deployments/
```suggestion
`doctl apps list-deployments {{app_id}}`
```